perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_user_attribute_def_virt_loa.java

```diff
@@ -0,0 +1,57 @@
+package cz.metacentrum.perun.core.impl.modules.attributes;
+
+import cz.metacentrum.perun.core.api.*;
+import cz.metacentrum.perun.core.api.exceptions.*;
+import cz.metacentrum.perun.core.impl.PerunSessionImpl;
+import cz.metacentrum.perun.core.implApi.modules.attributes.UserVirtualAttributesModuleAbstract;
+import cz.metacentrum.perun.core.implApi.modules.attributes.UserVirtualAttributesModuleImplApi;
+
+import java.util.List;
+
+/**
+ *
+ * @author Pavel Vyskocil vyskocilpavel@muni.cz
+ */
+public class urn_perun_user_attribute_def_virt_loa extends UserVirtualAttributesModuleAbstract  implements UserVirtualAttributesModuleImplApi {
+
+	public void checkAttributeValue(PerunSessionImpl sess, User user, Attribute attribute) throws InternalErrorException, WrongAttributeValueException, WrongReferenceAttributeValueException, WrongAttributeAssignmentException {
+		if(!attribute.equals(getAttributeValue(sess, user, attribute))) throw new WrongAttributeValueException(attribute, user, "Attribute value is not the highest value from user's UserExtSources Loas.");
+	}
+
+	public Attribute fillAttribute(PerunSessionImpl sess, User user, AttributeDefinition attributeDefinition) throws InternalErrorException, WrongAttributeAssignmentException {
+		return new Attribute(attributeDefinition);
+	}
+
+	public Attribute getAttributeValue(PerunSessionImpl sess, User user, AttributeDefinition attributeDefinition) throws InternalErrorException {
```

Comment:
I guess you only need to implement ``getAttributeDefinition()`` and ``getAttributeValue()`` since rest of methods has same implementation in parent abstract class. Maybe also ``checkAttributeValue()`` depending on @licehammer response on another comment.